### PR TITLE
Stubs

### PR DESCRIPTION
**Pull request by bwesterb**

### CODE REVIEW

File: draft-connolly-cfrg-xwing-kem.md [hunk 1]
```diff
@@ -117,50 +119,191 @@ informative:
...
+   aspects of X-Wing as compared to a more generic combiner.
+
+    - Simplicity of definition. For one, because all shared secrets
+       and cipher texts are fixed length, we do not need to encode the length.
+
+    - Its security analysis. Because ML-KEM-768 already assumes QROM, we
+       do not need to complicate the analysis of X-Wing by considering
+       weaker models.
+
+    - Its performance. By using SHA3-256 in the combiner, which matches
```
Comment (dconnolly) [inline draft-connolly-cfrg-xwing-kem.md:160]: sha3 also is more secure than sha2 and avoids requiring hkdf

File: draft-connolly-cfrg-xwing-kem.md [hunk 2]
```diff
@@ -117,50 +119,191 @@ informative:
...
+The aim of X-Wing is to provide a concrete, simple choice
+    for post-quantum hybrid KEM,
+    that should be suitable for the vast majority of use cases.
+
+## Design goals
+
+1. By making concrete choices, we can simplify and improve  many
+   aspects of X-Wing as compared to a more generic combiner.
+
+    - Simplicity of definition. For one, because all shared secrets
```
Comment (dconnolly) [inline draft-connolly-cfrg-xwing-kem.md:153]: 👍

File: draft-connolly-cfrg-xwing-kem.md [hunk 3]
```diff
@@ -117,50 +119,191 @@ informative:
...
+# Construction
+
+## Key derivation
+
+An X-Wing keypair (private key, public key) is derived from entropy
+as follows.
+
+~~~
+def DeriveKeyPair(ikm):
+  seed = SHAKE128(ikm, 96)
```
Comment (dconnolly) [inline draft-connolly-cfrg-xwing-kem.md:229]: `SHAKE` here, `SHA3-256` elsewhere, ...?

File: draft-connolly-cfrg-xwing-kem.md (hunk 1 above)
Comment (bwesterb) [reply draft-connolly-cfrg-xwing-kem.md]: Technically SHA3 is more secure, but it feels to me like stating that a 40,000kg rock is harder to lift by a person than a 10,000kg rock. Both are impossible.
The hkdf point is a good one.

**Comment by dconnolly** [comment]
Good to merge, we'll keep cranking on the TODOs

**Pull request merged**